docs/troubleshoot/security.md

```diff
@@ -4,7 +4,7 @@ Refer to vulerability management and release documentation [here](https://github
 
 ## Signature Validation
```

Comment:
Suggestion to move security out of troubleshoot :)

Reply:
Sure any recommendations on where to move it to?

Reply:
The current structure of Ratify docs is a bit chaos. It is supposed to have chapter for installation. Maybe we could put this guidance under the [Reference](https://ratify.dev/docs/1.2/category/reference) chapter first and add a link to the bottom of two installation guides [Getting Started](https://ratify.dev/docs/1.2/quick-start) and [Manual Quick Start](https://ratify.dev/docs/1.2/quickstarts/quickstart-manual).

Reply:
sounds good